examples/AppSwitcher/Macros.cpp

```diff
@@ -28,6 +28,7 @@ namespace H = kaleidoscope::hostos;
 static bool appSwitchActive = false;
 
 const macro_t *macroAppSwitch(uint8_t keyState) {
+  appSwitchActive = true;
```

Comment:
My first instinct was that this should be in the `keyIsPressed()` branch below, to not set it on key release. But in the end, that doesn't really matter, since cancel is done by a whole different key. I think this will be fine.